src/test/ut/ut_cortxfs_dir_ops.c

```diff
@@ -1146,6 +1255,9 @@ int main(void)
 			     create_remove_subdir_teardown),
 		ut_test_case(link_unlink_file, link_unlink_file_setup,
 			     link_unlink_file_teardown),
+		ut_test_case(delete_nonempty_dir, delete_nonempty_dir_setup,
+                             delete_nonempty_dir_teardown),
+		ut_test_case(delete_nonexistent_dir, NULL, NULL),
```

Comment:
Should we have happy path test case here ?
Test case for deleting empty directory & checking for its success.

Reply:
Added test case for deleting empty directory.